mbed_targets/_internal/target_database.py

```diff
@@ -18,7 +18,7 @@
 
 def get_target_database_path() -> pathlib.Path:
     """Return the path to the offline target database."""
-    return pathlib.Path(INTERNAL_PACKAGE_DIR, "data", "targets.json")
+    return pathlib.Path(INTERNAL_PACKAGE_DIR, "data", "offline_target_list.json")
```

Comment:
We'll also need to change the path in `sync_target_database` as it uses a path derived from `ConfigurationVariable.PROJECT_ROOT`... or we should fix `sync_target_database` so it uses this function.

Reply:
Ah cool, yes I'll take a look at that. I also liked your description in the last PR of that file being an 'offline database snapshot' so I might tweak the name too. 👍